### DISCUSSION TITLE
Heavy Equals Sign

### DISCUSSION THREAD

**Discussion by inyoka** [israel-dryer/ttkbootstrap#539] [Q&A] [answered]
Is there a reason there is no Heavy Equals Sign in the Emoji's?  It seems like there should be and I am thinking I must be missing something.  Many thanks.

**Comment by inyoka** [comment]
My employer suggested using a Calculator instead of an equals, unfortunately that emoji is also not available.  I understand these are not important issues, I'm just curious as to whether they'll be included. Thank you.

**Comment by israel-dryer** [comment, marked as answer]
The `Emoji` collection in 1.x is just a name->character lookup table — `Emoji.get("winking face")` returns the literal unicode character, nothing more. There's no rendering behind it; it drops the codepoint into the widget's `text`, and how it displays is up to your OS's emoji font. So the collection is only a curated snapshot of Unicode emoji, and a few glyphs — including **Heavy Equals Sign** (🟰) and anything "calculator" (the closest included is the abacus, 🧮) — just aren't in that fixed list.

Because it's only a veneer over the character, you don't need the table at all — paste the character directly:

```python
ttk.Button(root, text="🟰 Equals")
ttk.Button(root, text="🧮 Calc")
```

Heads-up for the future: in **2.0**, the `Emoji` collection is being removed — it was a thin wrapper over unicode characters that rendered inconsistently across platforms. It's replaced with proper, cross-platform **Bootstrap Icons** support (a bundled icon font rendered into theme-aware, recolorable glyphs). You can browse and search the full set — with visuals — at https://icons.getbootstrap.com/, which is the direction to use for iconography going forward.
